src/main/java/org/sagebionetworks/template/cdn/CdnBuilderImpl.java

```diff
@@ -72,6 +77,16 @@ VelocityContext createContext(Type type) {
 			ctxt.put(CTXT_KEY_ACM_CERT_ARN, acmCertificateArn);
 			String subDomain = Constants.isProd(stack) ? PROD_STACK_NAME : DEV_STACK_NAME;
 			ctxt.put(CTXT_KEY_SUBDOMAIN_NAME, subDomain);
+			try {
+				byte[] publicKey = dataCDNPublicKey.getBytes("UTF-8");
+				byte[] publicKeyMD5 = MessageDigest.getInstance("MD5").digest(publicKey);
```

Comment:
we can use DigestUtil.md5Hex(byte[] data) of apache.commons instead of line 82 and 83